pkgs/test/default.nix

```diff
@@ -93,4 +93,6 @@ with pkgs;
   };
 
   pkgs-lib = recurseIntoAttrs (import ../pkgs-lib/tests { inherit pkgs; });
+
+  nixpkgsCheckByName = callPackage ./nixpkgs-check-by-name { };
```

Comment:
I don't think the package should be hidden, and it should be named as a proper package in snake case. Could you add it to `all-packages.nix` or is that planned for PR 1b?

Reply:
It's not strictly hidden, you can build it using `nix-build -A tests.nixpkgsCheckByName`. But it's not meant to be installed by users or to be stable, so I don't think it should be in the main package set. In particular also, `tests` is not being recursed into by `nix search`/`nix-env`/search.nixos.org (`tests.recurseForDerivations` is not set), so it doesn't show up in searches.

Changing the attribute to `nixpkgs-check-by-name` though sounds okay.